draft-ietf-cbor-packed.md

```diff
@@ -882,8 +886,8 @@ from which the shared item is referenced.
 A regular shared item reference can be used to reference an
 integration tag.
 (Note that the generation of an integration tag can in turn be
-automatic in the table setup mechanism specified by a media type or a
-table setup tag, so the integration tag may never actually physically
+automatic in the table setup mechanism specified by an application environment
```

Comment:
This is the right word, but it may not be easy to make the connection to Section 3 where it is defined.

```suggestion
automatic in the table setup mechanism specified by an application environment ({{table-setup}})
```